packages/ts-transforms/test/matcher-spec.ts

```diff
@@ -74,4 +75,47 @@ describe('matcher', () => {
         expect(results.length).toEqual(1);
         expect(results[0].getMetadata('selectors')).toEqual(rules);
     });
+
+    it('matcher can run with notification rules', async () => {
+        const rules = [
+            'some:data AND bytes:>=1000',
+            'other:/.*abc.*/ OR _created:>=2018-11-16T15:16:09.076Z'
+        ];
+
+        const config: WatcherConfig = {
+            types: { _created: 'date' },
+            notification_rules: rules.join('\n')
+        };
+
+        const data = DataEntity.makeArray([
+            { some: 'data', bytes: 1200, _created: '2018-12-16T15:16:09.076Z' },
+            { some: 'data', bytes: 200 },
+            { some: 'other', bytes: 1200 }
+        ]);
+
+        const test = await opTest.init(config);
+        const results =  await test.run(data);
+        // each match will be inserted into the results
+        expect(results.length).toEqual(1);
+        expect(results[0].getMetadata('selectors')).toEqual(rules);
+    });
+
+    it('matcher throws with bad notification rules', async () => {
+        const rules1 = [
+            'some:data AND bytes:>=1000',
+            'alasdf{789q;lk]erasp98d7@'
+        ];
+
+        const config1: WatcherConfig = {
+            types: { _created: 'date' },
+            notification_rules: rules1.join('\n')
+        };
+
+        try {
+            await opTest.init(config1);
```

Comment:
If this passes, the test should fail, but currently this will pass since there is expectation.

```suggestion
            await expect(opTest.init(config1)).toReject();
```